pkg/controller.v1/pytorch/pytorchjob_controller.go

```diff
@@ -155,13 +156,18 @@ func (r *PyTorchJobReconciler) Reconcile(ctx context.Context, req ctrl.Request)
 	// Set default priorities to pytorch job
 	r.Scheme.Default(pytorchjob)
 
+	// parse volcano Queue from pytorchjob Annotation
+	schedulingPolicy := &commonv1.SchedulingPolicy{
```

Comment:
Since Pytorch spec embed `runPolicy`, can we get scheduling policy directly from `pytortchjob.Spec.RunPolicy.SchedulingPolicy`? 
 @qiankunli

Reply:
We should use `pytorchjob.Spec.RunPolicy` as the argument to reconcile the jobs

Reply:
https://github.com/kubeflow/tf-operator/blob/f9e5004ee41c923e3ba1f804a97116c84dfd5779/pkg/controller.v1/xgboost/xgboostjob_controller.go#L176

Reply:
@Jeffwan  

now it is always nil for SchedulingPolicy in pytorch-operator, if SchedulingPolicy is seted , it is ok use `pytortchjob.Spec.RunPolicy.SchedulingPolicy` directly

```go
// github.com/kubeflow/tf-operator/pkg/controller.v1/pytorch/pytorchjob_controller.go
runPolicy := &commonv1.RunPolicy{
		CleanPodPolicy:          pytorchjob.Spec.RunPolicy.CleanPodPolicy,
		TTLSecondsAfterFinished: pytorchjob.Spec.RunPolicy.TTLSecondsAfterFinished,
		ActiveDeadlineSeconds:   pytorchjob.Spec.RunPolicy.ActiveDeadlineSeconds,
		BackoffLimit:            pytorchjob.Spec.RunPolicy.BackoffLimit,
		SchedulingPolicy:        nil,
	}
// Use common to reconcile the job related pod and service
err = r.ReconcileJobs(pytorchjob, pytorchjob.Spec.PyTorchReplicaSpecs, pytorchjob.Status, runPolicy)
```

Reply:
@Jeffwan I update the pr 
```
	runPolicy := &commonv1.RunPolicy{
		CleanPodPolicy:          pytorchjob.Spec.RunPolicy.CleanPodPolicy,
		TTLSecondsAfterFinished: pytorchjob.Spec.RunPolicy.TTLSecondsAfterFinished,
		ActiveDeadlineSeconds:   pytorchjob.Spec.RunPolicy.ActiveDeadlineSeconds,
		BackoffLimit:            pytorchjob.Spec.RunPolicy.BackoffLimit,
		SchedulingPolicy:        pytorchjob.Spec.RunPolicy.SchedulingPolicy,
	}
```

Reply:
@qiankunli 

1. Can you help update this for MXNet job as well? 
2. Actually, since pytorch.Spec.RunPolicy is &commonv1.RunPolicy. We can pass `pytorchjob.Spec.RunPolicy` instead of constructing a new one.  See xgboost example

https://github.com/kubeflow/tf-operator/blob/acba15e644b4c4d4fe6b68664407e4ea588d4458/pkg/controller.v1/xgboost/xgboostjob_controller.go#L176

Reply:
> Can you help update this for MXNet job as well?

Should we make it in another PR?